ui/components/app/transaction-list/transaction-list.test.js

```diff
@@ -225,6 +242,24 @@ describe('TransactionList', () => {
     expect(container).toMatchSnapshot();
   });
 
+  it('renders TransactionList component with props hideTokenTransactions correctly', () => {
+    const defaultState2 = {
+      ...defaultState,
+      metamask: {
+        ...defaultState.metamask,
+        transactions: [MOCK_TRANSACTION_BY_TYPE[TransactionType.swap]],
+      },
+    };
+    const store = configureStore(defaultState2);
+    const { container } = renderWithProvider(
+      <MetaMetricsContext.Provider value={mockTrackEvent}>
+        <TransactionList hideTokenTransactions />
+      </MetaMetricsContext.Provider>,
+      store,
+    );
+    expect(container).toMatchSnapshot();
```

Comment:
Nit - do we want to test that the list is filtering tokens?

Reply:
OK nice, I do see that the snapshot correctly hides tokens - awesome work.
I think this can be a future task (move away from snapshot and target specific element to see if it exists)